App/en.lproj/Localizable.strings

```diff
@@ -578,4 +578,4 @@
 
 // MARK: Apple Watch
 "loading_instances_icloud" = "Loading instances from iCloud...";
-"watch_stuck_reinstall" = "This can take a while. If the app is stuck on this spinner, please try a reinstall of the app. Sorry for the inconvenience!";
+"watch_stuck_reinstall" = "This can take a while. If the app is stuck on a spinner, please try reinstalling it. Sorry for the inconvenience!";
```

Comment:
Maybe we should change 'this spinner' to 'this screen'?

Reply:
You can provide the screenshot in the situation.

Reply:
I thought about it and this screen is best I think. I'll change it to that :-)